FueledUtils/UIExtensions.swift

```diff
@@ -114,45 +150,124 @@ public extension UIView {
+	///
+	/// Refer to the documentation for `deselectAllRows(animated:)` for more info.
+	///
+	@available(*, deprecated, renamed: "deselectAllRows(animated:)")
 	public func deselectAllRows(_ animated: Bool) {
+		self.deselectAllRows(animated: animated)
+	}
+
+	///
+	/// Deselect all currently selected rows, optionally with an animation.
+	///
+	/// - Parameters:
+	///   - animated: `true` if you want to animate the deselection, and `false` if the change should be immediate.
+	///
+	public func deselectAllRows(animated: Bool) {
 		if let indexPaths = self.indexPathsForSelectedRows {
-			for indexPath in indexPaths {
-				self.deselectRow(at: indexPath, animated: animated)
-			}
+			self.deselectRows(indexPaths, animated: animated)
 		}
 	}
 }
 
 public extension UICollectionView {
+	///
+	/// Deselect given items, optionally with an animation.
+	///
+	/// - Parameters:
+	///   - indexPaths: The items to deselect.
+	///   - animated: `true` if you want to animate the deselection, and `false` if the change should be immediate.
+	///
+	public func deselectItems(_ indexPaths: [IndexPath], animated: Bool) {
+		for indexPath in indexPaths {
+			self.deselectItem(at: indexPath, animated: animated)
+		}
+	}
+
+	///
+	/// **Deprecated**: Please use `deselectAllItems(animated:)` instead.
+	///
+	/// Refer to the documentation for `deselectAllItems(animated:)` for more info.
+	///
+	@available(*, deprecated, renamed: "deselectAllItems(animated:)")
 	public func deselectAllItems(_ animated: Bool) {
+		self.deselectAllItems(animated: animated)
+	}
+
+	///
+	/// Deselect all currently selected rows, optionally with an animation.
+	///
+	/// - Parameters:
+	///   - animated: `true` if you want to animate the deselection, and `false` if the change should be immediate.
+	///
+	public func deselectAllItems(animated: Bool) {
 		if let indexPaths = self.indexPathsForSelectedItems {
-			for indexPath in indexPaths {
-				self.deselectItem(at: indexPath, animated: animated)
-			}
+			self.deselectItems(indexPaths, animated: animated)
 		}
 	}
 }
 
 public extension UIImage {
-	public static func draw(size: CGSize, opaque: Bool = false, scale: CGFloat = 0, graphics: (CGContext) -> Void) -> UIImage {
-		var image: UIImage?
+	///
+	/// Create a `CGContext` allowing to do custom drawing, and returns the resulting image as drawn in the context.
+	/// - Parameters:
+	///   - size: Please refer to the parameters documentation for `UIGraphicsBeginImageContextWithOptions` for more info.
+	///   - siopaqueze: Please refer to the parameters documentation for `UIGraphicsBeginImageContextWithOptions` for more info.
+	///   - scale: Please refer to the parameters documentation for `UIGraphicsBeginImageContextWithOptions` for more info.
+	///   - graphics: The drawing actions to execute.
+	///
+	public static func draw(size: CGSize, opaque: Bool = false, scale: CGFloat = 0.0, graphics: (CGContext) -> Void) -> UIImage {
```

Comment:
I want to update this method later on to return the `optional` it should return, for example if `size`'s `width` or `height` is 0.0, the resulting image will be `nil`. The user should be responsible for force-casting if he's sure the image is valid.